deploy/examples/keycloak/external-db-secret.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keycloak-db-secret
+  namespace: keycloak
+  labels:
+    app: keycloak
```

```suggestion
    app: sso
```
As in:
https://github.com/keycloak/keycloak-operator/blob/4ad1fc2e02ac026edb7604f8d0f13d0aa884e111/deploy/examples/keycloak/keycloak.yaml#L6